src/Compilers/Core/Portable/Emit/EditAndContinue/ILambdaHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+    internal interface ILambdaSyntaxHelper
```

Comment:
My concern was less about hte class being abstract/static and more about it being a bunch of static methods.  We'd have to go and put in a ton of instance methods that would then have to call the statics.  It feels like it would be pretty ugly :)